lightning/src/ln/channelmanager.rs

```diff
@@ -4597,8 +4367,8 @@ where
 	}
 
 	/// When a channel is removed, two things need to happen:
-	/// (a) [`ChannelManager::convert_channel_err`] must be called in the same `per_peer_state` lock as the
-	///     channel-closing action,
+	/// (a) The close error must be converted to [`MsgHandleErrInternal`] in the same `per_peer_state` lock as the
```

Comment:
Depending on if we rename the other methods from `convert` this might also be confusing. But this also implies that you can just build the `MsgHandleErrInternal` rather than calling one of the 3 close-handle-convert methods, maybe just link all of them?

Reply:
Linked all.